packages/collector/test/tracing/protocols/grpc/test.js

```diff
@@ -228,7 +228,9 @@ function checkTrace(serverControls, clientControls, spans, url, cancel, erroneou
   // Would be nice to also check for the log span from the interceptor but will actually never be created because at
   // that time, the parent span is an exit span (the GRPC exit). If only log spans were intermediate spans :-)
   // expectExactlyOneMatching(spans, checkLogSpanFromClientInterceptor.bind(null, httpEntry));
-  expectExactlyOneMatching(spans, checkLogSpanAfterGrpcExit(httpEntry, url, cancel, erroneous));
+  return retry(() => {
+    expectExactlyOneMatching(spans, checkLogSpanAfterGrpcExit(httpEntry, url, cancel, erroneous));
+  });
```

Comment:
I think this is wrong. If I read the code correctly, `checkTrace` is only ever called inside a retry (in `waitForTrace`), so this would be a duplicated retry, or rather, a retry in a retry.